dev-libs/hidapi/hidapi-0.9.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils multilib-minimal
+
+DESCRIPTION="A multi-platform library for USB and Bluetooth HID-Class devices"
+HOMEPAGE="https://github.com/libusb/hidapi"
+SRC_URI="https://github.com/libusb/${PN}/archive/${P}.tar.gz"
+
+LICENSE="|| ( BSD GPL-3 HIDAPI )"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+IUSE="doc fox static-libs"
+
+RDEPEND="
+	kernel_linux? (
+		virtual/libudev:0[${MULTILIB_USEDEP}]
+		virtual/libusb:1[${MULTILIB_USEDEP}]
+	)
+	kernel_FreeBSD? (
+		virtual/libusb:1[${MULTILIB_USEDEP}]
+	)
+"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )
+	virtual/pkgconfig
+	fox? ( x11-libs/fox )"
```

Comment:
fox not linked against? (I don't know.)

Reply:
From README.md \:
> HIDAPI also comes with a Test GUI. The Test GUI is cross-platform and uses Fox Toolkit http://www.fox-toolkit.org. It
> will build on every platform which HIDAPI supports. Since it relies on a 3rd party library, building it is optional but
> recommended because it is so useful when debugging hardware.

From configure.ac \:
> if test "x$testgui_enabled" != "xno"; then
>		# On Linux and FreeBSD platforms, use pkg-config to find fox.
>		PKG_CHECK_MODULES([fox], [fox17], [], [PKG_CHECK_MODULES([fox], [fox])]